content/en/monitors/types/process.md

```diff
@@ -43,10 +64,16 @@ After defining your search, a graph is displayed above the search inputs with an
 
 ### Set alert conditions
 
-* The count was `above`, `above or equal to`, `below`, or `below or equal to`
-* the threshold during the last `5 minutes`, `15 minutes`, `1 hour`, etc. or `custom` to set a value between 1 minute and 48 hours.
+- The count was `above`, `above or equal to`, `below`, or `below or equal to`
+- the threshold during the last `5 minutes`, `15 minutes`, `1 hour`, or larger. Additionally, you can use `custom` to set a value between 5 minutes and 24 hours.
+
+Use thresholds to set a numeric value for triggering an alert. Datadog has two types of notifications (alert and warning). Live Process Monitors recover automatically based on the alert or warning threshold.
+
+#### Best practices for timeframe selection
+
+Datadog recommends to pick the largest timeframe that you can support. This allows for additional fault tolerance in the case where an Agent becomes unavailable or encounters network issues.
 
-Use thresholds to set a numeric value for triggering an alert. Datadog has two types of notifications (alert and warning). Live process monitors recover automatically based on the alert or warning threshold.
+Live Process Monitors uses a [rolling time](https://docs.datadoghq.com/monitors/configuration/?tab=thresholdalert#evaluation-window) windows to evaluate process count, e.g. the monitor triggers when the alerting condition is met in the past X minutes. It’s recommended to not use evaluation windows shorter than 5 minutes to prevent the monitor from triggering if there is any sporadic network disruption between the Process Agent and Datadog.
```

Comment:
```suggestion
Live Process Monitors uses a [rolling time window](https://docs.datadoghq.com/monitors/configuration/?tab=thresholdalert#evaluation-window) to evaluate process count, e.g. the monitor triggers when the alerting condition is met in the past X minutes. It’s recommended to not use evaluation windows shorter than 5 minutes to prevent the monitor from triggering if there is any sporadic network disruption between the Process Agent and Datadog.
```